west.yml

```diff
@@ -183,7 +183,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: e0116b8eae46c2d90f2c1b23baef73bb082d00bd
+      revision: pull/193/head
```

Comment:
This does not look like an upstreamable change, could you use a hash here?

Reply:
It is not upstreamable- the relevant PR needs to be merged first

The reason this change is currently stalled is that we need to relocate the NXP HAL driver to RAM as well as the in tree ethernet driver code, and we do not want to make the relocation directive in the HAL dependent on this application defined KConfig.